static/app/views/profiling/profilesProvider.tsx

```diff
@@ -31,26 +27,26 @@ function fetchFlamegraphs(
     .then(([data]) => data);
 }
 
-function getTransactionId(input: Profiling.ProfileInput): string | null {
-  if (isSchema(input)) {
-    return input.metadata.transactionID;
-  }
-  if (isSentrySampledProfile(input)) {
-    return input.transaction.id;
-  }
-  return null;
-}
-
-interface FlamegraphViewProps {
-  children: React.ReactNode;
+function fetchContinuousProfileFlamegraph(
+  api: Client,
+  query: ContinuousProfileQueryParams,
+  projectSlug: Project['slug'],
+  orgSlug: Organization['slug']
+): Promise<Profiling.ProfileInput> {
+  return api
+    .requestPromise(`/organizations/${orgSlug}/profiling/chunks/`, {
+      method: 'GET',
+      query: {
+        ...query,
+        project: projectSlug,
+      },
+      includeAllArgs: true,
+    })
+    .then(([data]) => data.chunk);
```

Comment:
I guess. we dont really need includeAllArgs if we just access data afterwards. Is this some old copy paste relic?

Reply:
yes, gonna leave it in for now